### PR TITLE
Mo/tail with carousel

### PR DESCRIPTION
**Pull request by mozieblo2**

### CODE REVIEW

File: src/library/components/atoms/Tile/Tile.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LinkWrapper from '../LinkWrapper';
+
+const Tile = ({ linkProps, width, height, img }) => (
```
Comment (wmosienski) [inline src/library/components/atoms/Tile/Tile.jsx:5]: naming, if the image is required I would suggest "ImageTile" or sth like this

File: src/library/components/molecules/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        justifyContent: 'center',
+        alignItems: 'start',
+        flexWrap: 'nowrap',
+        flexShrink: 0,
+        flexGrow: 0,
+        overflow: 'hidden',
+        boxSizing: 'border-box',
+        border: '1px solid #FFFFFF',
+        position: 'relative',
+        transform: `translate3d(-${offset}px, 0px, 0px)`,
```
Comment (wmosienski) [inline src/library/components/molecules/Carousel/Carousel.jsx:21]: why not translate?

**Review by wmosienski — COMMENTED** [review]
good job! I left some stupid comments so you know I read your code :)

File: src/library/components/atoms/Tile/Tile.jsx (hunk 1 above)
Comment (mozieblo2) [reply src/library/components/atoms/Tile/Tile.jsx:5]: improved

File: src/library/components/molecules/Carousel/Carousel.jsx (hunk 2 above)
Comment (mozieblo2) [reply src/library/components/molecules/Carousel/Carousel.jsx:21]: ? ;>

File: src/stories/Carousel/Carousel_stories.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+);
+
+export const Basic = Template.bind({});
+Basic.args = {
+    label: 'Carousel',
+    linkProps: {
+        href: '/bevi',
+    },
+    width: 277,
+    height: 422
```
[the remark this answers is not in the document]
Comment (mozieblo2) [reply src/stories/Carousel/Carousel_stories.jsx:45]: removed :)

File: src/library/components/molecules/Carousel/Carousel.jsx (hunk 2 above)
Comment (mozieblo2) [reply src/library/components/molecules/Carousel/Carousel.jsx:21]: good point, improved 👍

**Review by wmosienski — APPROVED** [review]

**Pull request merged**